src/transformers/models/bart/modeling_bart.py

```diff
@@ -1159,19 +1254,28 @@ def forward(
         if labels is not None:
             use_cache = False
 
+        if input_ids is None and inputs_embeds is not None:
+            raise NotImplementedError(
+                f"Passing input embeddings is currently not supported for {self.__class__.__name__}"
+            )
+
```

Comment:
Why not? It doesn't seem to be different to other classes

Reply:
Impossible to get the `eos_mask` when `input_ids` are not passed...